apps/admin_api/priv/swagger/transaction_consumption/request_bodies.yaml

```
@@ -25,3 +25,17 @@ ConsumptionConfirmationRequestBody:
           - id
         example:
           id: txn_01cbfg5m2ee06kzm8tbysfmmw5
+
+ConsumptionCancellationRequestBody:
+  description: Cancel an owned pending consumption using the specified ID.
+  required: true
+  content:
+    application/vnd.omisego.v1+json:
+      schema:
+        properties:
+          id:
+            type: string
+        required:
+          - id
+        example:
+          id: txn_01cbfg5m2ee06kzm8tbysfmmw5
```

```suggestion
          id: txc_01cbfg5m2ee06kzm8tbysfmmw5
```


Also for the unchanged `ConsumptionConfirmationRequestBody` above.